flags/eigendaflags/cli.go

```diff
@@ -163,10 +181,96 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			EnvVars:  []string{withEnvPrefix(envPrefix, "PUT_RETRIES")},
 			Category: category,
 		},
+	},
+		v2Flags(envPrefix, category)...,
```

Comment:
think we should use a separate v2 category

Reply:
https://github.com/Layr-Labs/eigenda-proxy/pull/296